internal/exec/atmos.go

```diff
@@ -96,7 +96,7 @@ func ExecuteAtmosCmd() error {
 	// Process the selected command, stack and component
 	fmt.Println()
 	u.PrintMessageInColor(fmt.Sprintf(
-		"Executing command: atmos %s %s --stack %s", selectedCommand, selectedComponent, selectedStack),
```

Comment:
technically, we should be using `os.Args[0]` which is the command that was executed, which may not be `atmos`.